### PR TITLE
Analysis to view

### LINKED ISSUE #292: Replace term "Analyses" with "Views" in UI?
**Revisit this question after #157 & #286.**

The "Analyses" shown in the "Available analyses" section (ex. Histogram, Flame Chart) are Trace Compass "Views" not Trace Compass "Analyses". Replacing the term "Analyses" with "Views" in the Trace Viewer will keep language consistent with Trace Compass and reduce confusion.

| Trace Viewer Analyses | TC Analyses and Views |
|-------------------|--------------------------|
| image              | image | 

## Trace Compass Analyses and Views
My understanding is that the existence of "TC Analyses" and "TC Views" comes from the implementation:
The trace data transformation path is: **Trace events --> TC Analyses --> TC Views**.
* **Analyses:** Organization or transformation of data useful for trace-based debugging. Prepared for ease of use in visualisations (or even as input to other analyses?). Often data is from trace files, but it could also be from elsewhere (ex. variable names from the program's source code). 
* **Views:** Visualization of data from one or more analyses.

### Are "TC Analyses" an important concept for a Trace Viewer user?
It doesn't seem like it. Contributors/Developers need to know whether they're writing an Analysis or a View, but the user is only interacting with Views, right?

----

CC: @ssmagula

### PR DESCRIPTION
**Pull request by MatthewKhouzam**
Fixes #292

Analyses are an abstract concept of computation, and more importantly,
the dataproviders are shown, they have a one for one relation to views,
but a many to one relation to analyses, so views is more accurate too.

Signed-off-by: Matthew Khouzam <matthew.khouzam@ericsson.com>

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -27,7 +27,13 @@
             "name": "Start in Firefox",
             "reAttach": true,
             "webRoot": "${workspaceRoot}",
-            "url": "http://localhost:3000"
+            "url": "http://localhost:3000",
+            "pathMappings": [
+                {
+                    "url": "webpack:///home/ematkho/work/git/theia-trace-extension/packages/react-components/src/components/table-output-component.tsx",
```
Comment (bhufmann) [inline .vscode/launch.json:33]: remove. Should not be needed

**Review by bhufmann — CHANGES_REQUESTED** [review]
Can you have 2 PRs, one for the change of vertical alignment and the second one for the renaming of the analysis to views? Both things are unrelated and should be tracked differently.

File: packages/react-components/src/components/timegraph-output-component.tsx [hunk 2]
```diff
@@ -415,10 +419,15 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
                     if (currentStyle['height']) {
                         symbolSize = currentStyle['height'] * symbolSize;
                     }
+                    let vAlign = 'center';
+                    if (currentStyle['vertical-align']) {
+                        vAlign = currentStyle['vertical-align'];
+                    }
```
Comment (bhufmann) [inline packages/react-components/src/components/timegraph-output-component.tsx:425]: the vertical align change is already merged, right?

**Review by bhufmann — CHANGES_REQUESTED** [review]

File: packages/react-components/src/components/timegraph-output-component.tsx [hunk 3]
```diff
@@ -297,8 +297,12 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
                     if (currentStyle) {
                         const color = this.hexStringToNumber(currentStyle['background-color']);
                         let height = this.props.style.rowHeight * 0.8;
-                        if (currentStyle['height']) {
-                            height = currentStyle['height'] * height;
+                        const heightConst = currentStyle['height'];
+                        if (heightConst) {
+                            if (heightConst !== 1) {
+                                console.log(currentStyle);
```
[the remark this answers is not in the document]
Comment (MatthewKhouzam) [reply packages/react-components/src/components/timegraph-output-component.tsx]: Fixed!

**Review by bhufmann — APPROVED** [review]
Thanks! Looks good to me.

**Review by PatrickTasse — CHANGES_REQUESTED** [review]
Should be abandoned? This is a console logging patch

File: packages/react-components/src/components/timegraph-output-component.tsx [hunk 4]
```diff
@@ -297,9 +297,8 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
                     if (currentStyle) {
                         const color = this.hexStringToNumber(currentStyle['background-color']);
                         let height = this.props.style.rowHeight * 0.8;
-                        const heightConst = currentStyle['height'];
-                        if (heightConst) {
-                            height = heightConst * height;
+                        if (currentStyle['height']) {
+                            height = currentStyle['height'] * height;
```
Comment (PatrickTasse) [inline packages/react-components/src/components/timegraph-output-component.tsx]: Is this change intentional?

**Review by bhufmann — APPROVED** [review]
Thanks. Looks goo to me!

File: README.md [hunk 5]
```diff
@@ -135,7 +135,7 @@ Select the **Trace Viewer** icon to switch to the trace viewer.
 
 ![Trace Viewer](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/theia-trace-extension-trace-explorer-0.0.2.png)
 
-Now, you will see 3 sections: **Opened Traces**, **Available Analyses** and **Time Graph Tooltip**. Select the trace that you opened and the available analyses will be populated. Now you can select different analysis and the graphs will be added into a single container view for that trace.
+Now, you will see 3 sections: **Opened Traces**, **Available Views** and **Time Graph Tooltip**. Select the trace that you opened and the available views will be populated. Now you can select different analysis and the graphs will be added into a single container view for that trace.
```
Comment (PatrickTasse) [inline README.md:138]: analysis -> views

**Review by PatrickTasse — CHANGES_REQUESTED** [review]
Also need to update analysis/es -> views in:

trace-explorer-opened-traces-widget.tsx:299,301
trace-context-component:281

**Review by bhufmann — APPROVED** [review]
Looks good to me!

**Review by PatrickTasse — APPROVED** [review]

**Review by bhufmann — APPROVED** [review]

**Comment by MatthewKhouzam** [comment]
Thanks all!

**Pull request merged**